src/routes/CreateSafePage/components/SafeCreationProcess.tsx

```diff
@@ -255,6 +267,33 @@ function SafeCreationProcess(): ReactElement {
           }
         />
       )}
+      {showCouldNotLoadModal && newSafeAddress && (
+        <GenericModal
+          onClose={onCancel}
+          title="Unable to load the new Safe"
+          body={
+            <div>
+              <Paragraph>
+                We are currently unable to load the Safe but it was successfully created <br />
+                and can be found under the following address{' '}
+                <InlineEthHashInfo
+                  hash={newSafeAddress}
+                  shortenHash={8}
+                  showCopyBtn
+                  explorerUrl={getExplorerInfo(newSafeAddress)}
+                />
```

Comment:
Seems that it's not prefixed for some reason?

Reply:
Also I would show an un-shortened address in this case.

Reply:
👍  I've updated the PR description with the new look